### DISCUSSION TITLE
Editor Without Formatting - Plain Text Only

### DISCUSSION THREAD

**Discussion by greatkiwi** [Tencent/cherry-markdown#706] [Q&A]
How do I set the Cherry Editor box - into which you type the Markdown code - into a plain text only format (like a textarea).  In other words, I want to see the raw Markdown code that I entered - not some partially prettied up version of it. 

# Hello
* Item One
* Item Two

And just exactly like the "Write" TAB in the editor that I'm writing this comment in works.

Many thanks.

**Comment by RSS1102** [comment]
![image](https://github.com/Tencent/cherry-markdown/assets/81673017/2f9b646a-f5e1-45ab-9edb-842a089633b9)
https://tencent.github.io/cherry-markdown/examples/index.html
You can understand that this is just a `tab`, where the `items `of both tabs are displayed. All you need to do is save the markdown str and display it when needed.

**Comment by greatkiwi** [comment]
Thanks for your prompt response.

This GitHub Markdown editing box is **ALMOST** what I was looking for.  Because it DOESN'T style the text in the editing box as you type.

The goal is that the text in the editing box should be raw (Gruber like) text, that could be sent as is - as the TEXT content of some email (and so that the recipient would still have a readable result).

In this Gruber like text mode, as well as the editor styling being switched off, you'd also have to use:-

Setext Headings
===============
Instead of ATX.  And a;

Fixed-Width Font
================
So that you can:-

* Easily get the correct number of "=" and "-" under the H1 and H2 headings.  and;

* Align TABLES correctly.  Eg;-
	| Column 1 | Column Two |
	|----------|------------|
	|123       |  456 	 |

But the GitHub editor uses a fixed width font.  So it's not perfect for creating text emails.  But switching to a fixed-width font shouldn't be too hard to so.

---

I'm not sure if the Cherry Markdown editor could be given this style-free, fixed width font mode (presumably, the resulting editor wouldn't need or use CodeMirror).  

But it might be useful, if it's a minor change that's easily done.

---

But don't do that for me.  I've decided that a better approach is to leave the Cherry Editor as it is.  And then convert the Cherry Editor Markdown to the Gruber-like TEXT format.
